app/javascript/components/works/Works.jsx

```diff
@@ -106,7 +147,22 @@ class Works extends React.Component {
           <div className="col-list-3">
             {works.slice(workStartIndex, workEndIndex).map((work, i) => {
               return (
-                <WorkColumnPanel key={i} work={work} />
+                <div>
+                  <WorkColumnPanel key={i} work={work}>
+                      {work.hidden == false && this.props.userType == "admin" &&
+                      <Button className="ml2" type="hover-button" onClick={() => this.hideWork(work.id)}>
```

Comment:
Only one button here so we don't need margin-left~
```suggestion
                      <Button type="hover-button" onClick={() => this.hideWork(work.id)}>
```